test/defaults.test.js

```diff
@@ -76,4 +76,34 @@ describe('defaults', function() {
       });
     });
   });
+
+  context('applyDefaultOnWrites', function() {
+    it('if not set, should not affect default behavior', function(done) {
```

Comment:
- Our test names are usually following the convention "subject, verb/effect, condition".
- We use arrow functions in new tests and async/await.

```suggestion
    it('does not affect default behavior when not set', async () => {
```